Plugin/DotpayDirectPlugin.php

```diff
@@ -143,25 +161,36 @@ public function createDotpayRedirectActionException(FinancialTransactionInterfac
         $instruction = $transaction->getPayment()->getPaymentInstruction();
 
         $extendedData = $transaction->getExtendedData();
-        $urlc         = $this->router->generate('ets_payment_dotpay_callback_urlc', array(
-            'id' => $instruction->getId()
-        ), true);
+        $urlc         = $this->router->generate(
+            'ets_payment_dotpay_callback_urlc',
+            ['id' => $instruction->getId()],
+            true
+        );
 
-        $datas = array(
+        $datas = [
```

Comment:
please fix typo everywhere in this file:
```suggestion
        $data = [
```